Parser/Python.asdl

```diff
@@ -3,14 +3,11 @@
 
 module Python
 {
-    mod = Module(stmt* body, type_ignore *type_ignores)
+    mod = Module(stmt* body, type_ignore* type_ignores)
```

Comment:
I don't know the syntax of this change. Is this change related to the removal of Suite? If not, please revert this change. If you want to push the change anyway, open a separated PR for it :-)

Reply:
I thouht it would be less noise if I push this optional styling with this PR.

Reply:
I will not block this PR just for this. It was just a remark. I wrote it before seeing that multiple other core devs already approved the PR. I don't understand if "*" applies to type_ignore or type_ignoes.

Reply:
This is an optional thing, but all usages in the file (and the examples in the ASDL specification) puts type mark (`*`/`?`) after the field type. This version already can be parse-able with the ASDL parser of the CPython but it makes things hard for outside parsers.

Reply:
I see. Let's keep your change in this PR.